### DISCUSSION TITLE
Donation receipt?

### DISCUSSION THREAD

**Discussion by AxelBorg** [transmission/transmission#4016] [Q&A] [answered]
Hi. If I donate to Transmission, can I get a receipt or something for my accounting dept...? Or is there another way?

**Comment by ckerr** [comment, marked as answer]
Hmm, we don't really get that many donations so the project's never needed to work up any receipt paperwork before.

You should be able to get a copy of your donation from PayPal, right? When I go into my PayPal account I can see where I've sent money in the past
